extensions/default/src/DicomWebDataSource/utils/clientManager.ts

```diff
@@ -0,0 +1,215 @@
+import { api } from 'dicomweb-client';
+import StaticWadoClient from './StaticWadoClient';
+import dcm4cheeReject from '../dcm4cheeReject';
+import { errorHandler, utils } from '@ohif/core';
+
+export default class ClientManager {
```

Comment:
whoever taking over this PR, please add docs above this

Reply:
docs?